packages/ia-activity-indicator/src/ia-activity-indicator.js

```diff
@@ -0,0 +1,162 @@
+
+const IAActivityIndicatorMode = Object.freeze({
+  processing: 'processing',
+  complete: 'complete',
+});
+
+// eslint-disable-next-line import/prefer-default-export
+class IAActivityIndicator extends LitElement {
+  static get properties() {
+    return {
+      mode: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.mode = IAActivityIndicatorMode.processing;
+  }
+
+  render() {
+    /* eslint-disable */
+    return html`
+      <div class="${this.mode}">
+        <svg
+          viewBox="0 0 120 120"
+          preserveAspectRatio="none"
+          version="1.1"
+          xmlns="http://www.w3.org/2000/svg"
+          xmlns:xlink="http://www.w3.org/1999/xlink"
+        >
+          <g
+            id="icons/check-ring---squared"
+            stroke="none"
+            stroke-width="1"
+            fill="none"
+            fill-rule="evenodd"
+          >
+            <path
+              id="completed-ring"
+              class="loaded-indicator"
+              d="M60,10 C70.5816709,10 80.3955961,13.2871104 88.4763646,18.8959201 L78.3502633,29.0214223 C72.9767592,25.8315427 66.7022695,24 60,24 C40.117749,24 24,40.117749 24,60 C24,79.882251 40.117749,96 60,96 C79.882251,96 96,79.882251 96,60 L95.995,59.46 L108.327675,47.128668 C109.350926,50.9806166 109.925886,55.015198 109.993301,59.1731586 L110,60 C110,87.6142375 87.6142375,110 60,110 C32.3857625,110 10,87.6142375 10,60 C10,32.3857625 32.3857625,10 60,10 Z"
+            ></path>
+            <polygon
+              id="check"
+              class="loaded-indicator"
+              transform="translate(75.000000, 41.500000) rotate(44.000000) translate(-75.000000, -41.500000) "
+              points="96 85 54 85 54 65 76 64.999 76 -2 96 -2"
+            ></polygon>
+            <path
+              id="activity-ring"
+              class="activity-indicator"
+              d="M60,10 C69.8019971,10 78.9452178,12.8205573 86.6623125,17.6943223 L76.4086287,27.9484118 C71.4880919,25.4243078 65.9103784,24 60,24 C40.117749,24 24,40.117749 24,60 C24,79.882251 40.117749,96 60,96 C79.882251,96 96,79.882251 96,60 C96,53.3014663 94.1704984,47.0302355 90.9839104,41.6587228 L101.110332,31.5326452 C106.715332,39.6116982 110,49.4222615 110,60 C110,87.6142375 87.6142375,110 60,110 C32.3857625,110 10,87.6142375 10,60 C10,32.3857625 32.3857625,10 60,10 Z"
+            ></path>
+            <g
+              id="activity-dots"
+              class="activity-indicator"
+              transform="translate(40.000000, 55.000000)"
+            >
+              <circle id="left-dot" cx="5" cy="5" r="5"></circle>
+              <circle id="middle-dot" cx="20" cy="5" r="5"></circle>
+              <circle id="right-dot" cx="35" cy="5" r="5"></circle>
+            </g>
+          </g>
+        </svg>
+      </div>
+    `;
+    /* eslint-enable */
+  }
+
+  static get styles() {
+    const checkmarkColorCss = css`var(--activityIndicatorCheckmarkColor, #31A481)`;
```

Comment:
Couldn't these `var` values be added directly to the CSS template literal?

Reply:
I put them out here so you can distinguish your CSS variables. Otherwise they get lost in the CSS.